### DISCUSSION TITLE
Open source start up questions.

### DISCUSSION THREAD

**Discussion by AllanRegush** [EddieHubCommunity/support#253] [Q&A]
1. What is stopping  someone from stealing the intellectual property or using the project without paying for the service?
2. Is getting funding the same as a closed source start up?

**Comment by eddiejaoude** [comment]
Thanks for the question @AllanRegush , here are my thoughts:

> 1.  What is stopping someone from stealing the intellectual property or using the project without paying for the service?

Nothing, but I really don't think people will steal it, because they have their own ideas. I can't remember who said this, but it really stuck in my head "no one will steal your stupid ideas, because they have their own stupid ideas" 😂 

There are different licenses you can use to be more strict or more open. http://choosealicense.com by GitHub is a great place to look. `GNU GPLv3 ` might be relevant to you:

```
The GNU GPLv3 also lets people do almost anything they want with your project, except distributing closed source versions.
```

> 2. Is getting funding the same as a closed source start up?

I did hear some places that offer funding get nervous about open source projects, it being open source and using open source software - but most of the internet is built on open source, so I think those people are going extinct 🦖 . TBH I don't know enough about this space to comment any further. I can try to find out more on this.
